app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h1>Here is a list of all posts and comments for a specific user!</h1>
\ No newline at end of file
+<div class="container">
+  <div class='user-info-container'>
+    <p class="user-photo">
+      <%= image_tag @user.photo %>
+    </p>  
+    <div class='user-info'>  
+      <p class="user-name"><%= link_to @user.name, user_path(@user) %></p>
+      <p class="posts-counter">Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+  <% @posts.each do |post| %>
+    <div class="post-info">
+      <p>Post #<%= post.id %></p>
+      <p><%= link_to post.text.truncate(100), user_post_path(post.author, post) %></p>
+      <div class="likes-comments">
+        <p>Comments: <%= post.comments_counter %></p>
+        <p>Likes: <%= post.likes_counter %></p>
+      </div>
+      <div class="comments-section<%= ' with-border' if post.comments_counter > 0 %>">
+        <% post.comments.each do |comment| %>
+          <div>
+            <p><%= comment.author.name %>: <%= comment.text %></p>
```

Comment:
- [ ] Kindly use the `recent_comments` post-model method you created to fetch 5 recent comments. You have to use these methods in the views because if you use them in the controllers, it will be difficult to fetch other comments in the future.